examples/basic-datasource/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM grafana/grafana:8
```

Comment:
I think we briefly discussed that the docker tags work like this. I get an error when I try using it:

```
 docker-compose up -d
Creating network "basic-datasource_default" with the default driver
Building grafana
[+] Building 1.9s (4/4) FINISHED                                                                                                                                        
 => [internal] load build definition from Dockerfile                                                                                                               0.2s
 => => transferring dockerfile: 674B                                                                                                                               0.0s
 => [internal] load .dockerignore                                                                                                                                  0.2s
 => => transferring context: 2B                                                                                                                                    0.0s
 => ERROR [internal] load metadata for docker.io/grafana/grafana:8                                                                                                 1.6s
 => [auth] grafana/grafana:pull token for registry-1.docker.io                                                                                                     0.0s
------
 > [internal] load metadata for docker.io/grafana/grafana:8:
------
failed to solve with frontend dockerfile.v0: failed to create LLB definition: docker.io/grafana/grafana:8: not found
ERROR: Service 'grafana' failed to build : Build failed
```